server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -368,4 +369,13 @@ private static Object handleWatchFailure(Throwable thrown) {
         }
         return converter.apply(Iterables.getOnlyElement(collection));
     }
+
+    @Get("/projects/{projectName}/repos/{repoName}/aggregation")
+    public CompletableFuture<?> aggregateEntries(@Param("revision") @Default("-1") String revision,
+                                                 @RequestObject Repository repository,
+                                                 @RequestObject(PathsRequestConverter.class)
+                                                                Iterable<String> paths) {
```

Comment:
Can we make `I` in `Iterable` placed in the same column with `@` above?

Reply:
Fixed. Thank you